modules/images/regenerate-existing-images/load.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * Module Name: Regenarate Existing Images
+ * Description: Regenerate existing images when the image sizes changes during plugin/theme
+ * activation/deactivation or for any other reasons.
```

Comment:
Please don't break this across multiple lines, as it doesn't work with the module header parsing. Also, suggesting a few tweaks to the description.
```suggestion
 * Description: Introduces background process infrastructure to regenerate existing image sizes and MIME types on demand.
```